pkgs/tools/package-management/nix-du/default.nix

```diff
@@ -27,13 +27,13 @@ rustPlatform.buildRustPackage rec {
 
   doCheck = true;
   nativeCheckInputs = [
-    nix
+    nixForLinking
```

Comment:
shouldn't this stay `nix`?  `nativeCheckInputs` aren't linked against.  We're calling the nix cli here; not linking against its library